services/minespace-web/src/components/Forms/tailing/tailingsStorageFacility/BasicInformation.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Col, Row, Typography } from "antd";
+import { Field } from "redux-form";
+import { maxLength, requiredList } from "@common/utils/Validate";
+import { renderConfig } from "@/components/common/config";
+
+const propTypes = {};
+
+export const BasicInformation = () => (
+  <>
+    <Typography.Title level={3}>Basic Information</Typography.Title>
+    <Field
+      id="facility_type"
+      name="facility_type"
+      label="Facility Type"
+      component={renderConfig.SELECT}
+      validate={[requiredList]}
+    />
+    <Field
+      id="tailings_storage_facility_type"
+      name="proponent_project_id"
+      label="Tailings Storage Facility Type"
+      component={renderConfig.FIELD}
+      validate={[maxLength(300)]}
+    />
+    <Field
+      id="mines_act_permit_number"
+      name="mines_act_permit_number"
+      label="Mines Act Permit Number"
+      component={renderConfig.FIELD}
+      validate={[maxLength(300)]}
+    />
+    <Field
+      id="mine_tailings_storage_facility_name"
+      name="mine_tailings_storage_facility_name"
+      label="Tailing Storage Facility Name"
+      component={renderConfig.FIELD}
+      validate={[maxLength(300)]}
+    />
+    <Row gutter={16}>
+      <Col span={12}>
+        <Field
+          id="latitude"
```

Comment:
More of a general question, are there any validation for lat/long other places in the app? If so, we should probably match it up

Reply:
Ah yes.  Indeed there is a validator for that.  Will add it in.